test/image.bats

```diff
@@ -365,3 +366,63 @@ EOF
 	expected_output=$(date +"%a %b %e %H:%M:%S %Z %Y")
 	[[ "$output" == *"$expected_output"* ]]
 }
+
+@test "run container with memory_limit_in_bytes -1" {
+	cat << EOF > "$CRIO_CONFIG_DIR/99-mem.conf"
+[crio.runtime]
+default_runtime = "mem"
+
+[crio.runtime.runtimes.mem]
+runtime_path = "/usr/local/bin/runc"
+EOF
+	start_crio
+	case $(go env GOARCH) in
+	amd64)
+		crictl pull ${IMAGE_LIST_DIGEST_FOR_TAG_AMD64}
+		IMAGE_MEM=${IMAGE_LIST_DIGEST_FOR_TAG_AMD64}
+		;;
+	arm64)
+		crictl pull ${IMAGE_LIST_DIGEST_FOR_TAG_ARM64}
+		IMAGE_MEM=${IMAGE_LIST_DIGEST_FOR_TAG_ARM64}
+		;;
+	esac
+
+	jq '.metadata.name = "memory"
+		|.command = ["/bin/sh", "-c", "sleep 600"]
+		|.linux.resources.memory_limit_in_bytes = -1
+		|.image.image = "'"$IMAGE_MEM"'"' \
```

Comment:
You can pass an argument to `jq`, to avoid having to deal with brackets and such.